### PR TITLE
Feature: Groups of user

### LINKED ISSUE #32: Display groups of the user
## Description

blocked by #15 
blocked by #13

On UserPage, add UserGroupsTable with pagination to display all group of the user.

Actions are not to implemented in this task.

image

### PR DESCRIPTION
**Pull request by justinehell**
Resolves #32

### CODE REVIEW

File: src/pages/UserPage.vue [hunk 1]
```diff
@@ -68,7 +84,18 @@ async function loadUser() {
     });
 }
 
-onMounted(() => {
+/**
+ * Get user groups using user Id.
+ * @returns {Promise<void>} Promise with nothing on success.
+ */
+async function search() {
```
Comment (Zorin95670) [inline src/pages/UserPage.vue:91]: rename to loadGroups
And add a new function search:
```js
function search() {
  loading.value = true;
  Promise.allSettled(
    loadUser(),
    loadGroups(),
  ).finaly(() => {
     loading.value = false;
  })
}
```

File: changelog.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   * Add action to redirect on user page.
   * Add action to delete a user.
 * Setup User page with:
-  * Display all information of a user.
+  * Display all information and groups of a user.
```
Comment (Zorin95670) [inline changelog.md:20]: make two sentence because we will have more functionnality

File: src/pages/UserPage.vue (hunk 1 above)
Comment (justinehell) [reply src/pages/UserPage.vue]: done

File: changelog.md (hunk 2 above)
Comment (justinehell) [reply changelog.md]: done

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**